src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -57,31 +57,54 @@ public static ConversationStore getTestInstance(PersistentStorageAgent persisten
 
   /** The in-memory list of Conversations. */
   private List<Conversation> conversations;
-	
-	/** The in-memory activity feed conversation. */
-	private Conversation actFeedConversation;
+
+  /** The in-memory activity feed conversation. */
+  private Conversation actFeedConversation;
+
+  /** The in-memory list of restricted conversation names reserved for private conversations. */
+  private List<String> restrictedConversationNames;
```

Comment:
Can this be final? And can you init it with the declaration? The constructor isn't really adding any value.. Same for conversations